draft-trammell-quic-spin.md

```diff
@@ -710,6 +710,41 @@ are actually within the detectable range.
   (both) the signal being detected and govern the non-uniform sampling
   frequency.
 
+# Greasing
+
+Routes, congestion levels and therefore latency between two fixed QUIC
+endpoints, as well as the shape of individual application flows, fluctuate in
+ways that are not totally predictable by an on path observer.  In general,
+there is no a-priori pattern for the spin-bit distribution that will always
+materialise on a certain flow aggregate, even for a single user.
+
+There has been discussion in the QUIC working group that greasing could be a
+strategy to counter an evil access provider that might gate access to its users
+on a valid spin bit signal.  Let's accept for a moment this threat model and
+consider the practical case of a home gateway that temporarily misbehaves, for
+example draining its queues slower than it'd normally do while a firmware
+download is in progress.  It would be ill-considered for an access provider
+(even a malicious one) to block, or otherwise interfere with, QUIC flows
+originating from behind that CPE solely based on the fact that RTTs are now
+different from "usual".  And that simply because of the sheer number of false
+negatives that the operator would get by setting an arbitrary threshold.
+
+So, the potential for ossification appears to be extremely low - in a sense,
+since it depends on so much external noise, the spin-bit is self-greasing.  In
```

Comment:
s/spin-bit/spin-bit result variability/
s/self-greasing/self-greasing to an extent/